pages/api/csv-export.js

```diff
@@ -23,19 +23,20 @@ async function getJamData(jamId) {
     .doc(jamId)
     .get()
     .then((doc) => {
-      console.log(doc.data());
       return doc.data();
     });
 }
 
 function createTitleFile(title) {
-  const jamName = _.truncate(_.snakeCase(title), { length: 30 });
-  const exportTime =
-    _.snakeCase(new Date().toLocaleDateString()) +
-    '_' +
-    _.snakeCase(new Date().toLocaleTimeString());
+  const putATimeStampOn = new Date();
+  const jamName = truncate(snakeCase(title), { length: 30 });
+  const timeStamp = snakeCase(
+    putATimeStampOn.toLocaleDateString() +
+      '_' +
+      snakeCase(putATimeStampOn.toLocaleTimeString()),
```

Comment:
Sorry if I'm missing something, isnt it just the same date repeated? so it'd be `2020-01-01_2020-01-01`?

Reply:
<img width="512" alt="image" src="https://user-images.githubusercontent.com/81761225/131872054-1ddf2095-293c-4b01-9f60-2424d3d7fff7.png">

It should put a title, current date and current time. Is that how you understood it?

Reply:
Ah one is `Date` and one is `Time`! Sorry I was misreading it as the same string twice!